.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+test:default:
+  variables:
+    CMAKE_BUILD_TYPE: "Release"
+  image: votca/buildenv
+  cache:
+    paths:
+      - /home/votca/.ccache/
+  script:
+    - mktexfmt latex.fmt
+    - ccache -z
+    - j="$(grep -c processor /proc/cpuinfo 2>/dev/null)" || j=0; ((j++))
+    - git branch -D current_commit
```

Comment:
@onurcaylak if I don't remove the branch first, I am getting an error that the branch already exists, which is strange! I thought the changes in the git repository doesn't get cached! Any idea?

Reply:
Still no sure....